PackageVersionsGeneratorDefinitions.json

```diff
@@ -1,59 +1,67 @@
-    },
-    {
-        "IntegrationName": "SqlClient",
-        "SampleProjectName": "Samples.SqlServer",
-        "NugetPackageSearchName": "System.Data.SqlClient",
-        "MinVersion": "4.1.0",
-        "MaxVersionExclusive": "5.0.0"
-    },
-    {
-        "IntegrationName": "StackExchangeRedis",
-        "SampleProjectName": "Samples.StackExchange.Redis",
-        "NugetPackageSearchName": "StackExchange.Redis",
-        "MinVersion": "1.0.187",
-        "MaxVersionExclusive": "2.0.0"
-    },
-    {
-        "IntegrationName": "ServiceStackRedis",
-        "SampleProjectName": "Samples.ServiceStack.Redis",
-        "NugetPackageSearchName": "ServiceStack.Redis",
-        "MinVersion": "4.0.48",
-        "MaxVersionExclusive": "6.0.0"
-    },
-    {
-        "IntegrationName": "AspNetCoreMvc2",
-        "SampleProjectName": "Samples.AspNetCoreMvc2",
-        "SampleTargetFramework": "netcoreapp2.1",
-        "NugetPackageSearchName": "Microsoft.AspNetCore.Mvc",
-        "MinVersion": "2.0.0",
-        "MaxVersionExclusive": "3.0.0"
-    }
+  {
+    "IntegrationName": "MongoDB",
+    "SampleProjectName": "Samples.MongoDB",
+    "NugetPackageSearchName": "MongoDB.Driver",
+    "MinVersion": "2.0.0",
+    "MaxVersionExclusive": "3.0.0"
+  },
+  {
+    "IntegrationName": "ElasticSearch6",
+    "SampleProjectName": "Samples.Elasticsearch",
+    "NugetPackageSearchName": "Elasticsearch.Net",
+    "MinVersion": "6.0.0",
+    "MaxVersionExclusive": "7.0.0"
+  },
+  {
+    "IntegrationName": "ElasticSearch5",
+    "SampleProjectName": "Samples.Elasticsearch.V5",
+    "NugetPackageSearchName": "Elasticsearch.Net",
+    "MinVersion": "5.3.0",
+    "MaxVersionExclusive": "6.0.0"
+  },
+  {
+    "IntegrationName": "Npgsql",
+    "SampleProjectName": "Samples.Npgsql",
+    "NugetPackageSearchName": "Npgsql",
+    "MinVersion": "4.0.0",
+    "MaxVersionExclusive": "5.0.0"
+  },
+  {
+    "IntegrationName": "SqlClient",
+    "SampleProjectName": "Samples.SqlServer",
+    "NugetPackageSearchName": "System.Data.SqlClient",
+    "MinVersion": "4.1.0",
+    "MaxVersionExclusive": "5.0.0"
+  },
+  {
+    "IntegrationName": "StackExchangeRedis",
+    "SampleProjectName": "Samples.StackExchange.Redis",
+    "NugetPackageSearchName": "StackExchange.Redis",
+    "MinVersion": "1.0.187",
+    "MaxVersionExclusive": "2.0.0"
+  },
+  {
+    "IntegrationName": "ServiceStackRedis",
+    "SampleProjectName": "Samples.ServiceStack.Redis",
+    "NugetPackageSearchName": "ServiceStack.Redis",
+    "MinVersion": "4.0.48",
+    "MaxVersionExclusive": "6.0.0"
+  },
+  {
+    "IntegrationName": "AspNetCoreMvc2",
+    "SampleProjectName": "Samples.AspNetCoreMvc2",
+    "SampleTargetFramework": "netcoreapp2.1",
+    "NugetPackageSearchName": "Microsoft.AspNetCore.Mvc",
+    "MinVersion": "2.0.0",
+    "MaxVersionExclusive": "3.0.0"
+  },
+  {
+    "IntegrationName": "AmazonSqs",
+    "SampleProjectName": "Samples.Amazon.SQS",
+    "SampleTargetFramework": "netcoreapp2.1",
```

Comment:
Can you remove this `SampleTargetFramework` property? This will limit it to only running on 2.1 and not 3.0